experimental/dds/tree2/src/typed-tree/typedTree.ts

```diff
@@ -82,31 +77,16 @@ export class TypedTreeFactory<TRoot extends FieldSchema = FieldSchema> implement
 		id: string,
 		services: IChannelServices,
 		channelAttributes: Readonly<IChannelAttributes>,
-	): Promise<TypedTreeChannel<TRoot>> {
+	): Promise<TypedTreeChannel> {
 		const tree = new SharedTree(id, runtime, channelAttributes, this.options, "SharedTree");
 		await tree.load(services);
-		return this.prepareChannel(runtime, tree);
+		return new ChannelWrapperWithSchematize(runtime, tree);
 	}
 
-	public create(runtime: IFluidDataStoreRuntime, id: string): TypedTreeChannel<TRoot> {
+	public create(runtime: IFluidDataStoreRuntime, id: string): TypedTreeChannel {
 		const tree = new SharedTree(id, runtime, this.attributes, this.options, "SharedTree");
 		tree.initializeLocal();
-		// TODO: Once various issues with schema editing are fixed separate initialize from schematize, do initialize here
```

Comment:
With the late schema binding, this is no longer possible since we don't have the initial content until after the create and load code paths have converged. Thus this TODO is removed.